crates/cli/Cargo.toml

```diff
@@ -9,9 +10,9 @@ description = "Trdelník is Rust based testing framework providing several conve
 
 [dependencies]
 clap = { version = "3.1.6", features = ["derive"] }
-tokio = { version = "1.17.0", features = ["rt-multi-thread", "macros", "fs", "signal", "sync", "time", "io-util", "process"], default-features = false }
+tokio = { version = "~1.14.1", features = ["rt-multi-thread", "macros", "fs", "signal", "sync", "time", "io-util", "process"], default-features = false }
 anyhow = { version = "1.0.56", features = ["std"], default-features = false }
 fehler = { version = "1.0.0", default-features = false }
-solana-sdk = "~1.10.39"
+solana-sdk = "=1.10.29"
```

Comment:
Aww... this looks scary!

Reply:
That's the latest version we can bump to because of Anchor